native/wasmex/src/wit.rs

```diff
@@ -0,0 +1,18 @@
+use rustler::{NifResult, Term};
+use wit_parser::{Resolve, WorldItem};
+
+#[rustler::nif(name = "wit_exported_functions")]
+pub fn exported_functions(env: rustler::Env, path: String, wit: String) -> NifResult<Term> {
```

Comment:
I know you care only about functions for this PR, but is it an option to returns all exports of the module instead?

similar to [`module_exports`](https://github.com/tessi/wasmex/blob/1351e6dd0056452973f94611d8d815fbb13d63b1/native/wasmex/src/module.rs#L62) for regular wasm modules.

We would share a common/similar API (which I like for consistency) and having that additional info will surely help us later and we don't have to break any API for those changes.

Reply:
I'll take a look at module exports and see if we can consolidate, agree having a consistent api would be best.